stable_audio_tools/interface/gradio.py

```diff
@@ -183,7 +187,7 @@ def progress_callback(callback_info):
 
     # Convert to WAV file
     audio = rearrange(audio, "b d n -> d (b n)")
-    audio = audio.clamp(-1, 1).mul(32767).to(torch.int16).cpu()
+    audio = audio.to(torch.float32).div(torch.max(torch.abs(audio))).clamp(-1, 1).mul(32767).to(torch.int16).cpu()
```

Comment:
We could probably put this transformation into a utility function

Reply:
yeah, I'd rather do that in the dev repo and bring it over later